src/Catalog/Monitoring/ValidationTestCollector.cs

```diff
@@ -0,0 +1,76 @@
+        {
+            _auditingStorage = auditingStorage;
+            _packageValidationTestRunner = packageValidationTestRunner;
+            _logger = logger;
+        }
+        
+        public IEnumerable<PackageValidationTestResult> PackageValidationResults => new List<PackageValidationTestResult>();
+
+        protected override async Task ProcessSortedBatch(CollectorHttpClient client, KeyValuePair<Tuple<string, string>, IList<JObject>> sortedBatch, JToken context, CancellationToken cancellationToken)
+        {
+            var packageId = sortedBatch.Key.Item1;
+            var packageVersion = sortedBatch.Key.Item2;
+            var catalogEntriesJson = sortedBatch.Value;
+
+            try
+            {
+                var testResults = await _packageValidationTestRunner.Run(packageId, packageVersion, catalogEntriesJson, _auditingStorage, client, cancellationToken, _logger);
+                _logger.LogInformation("Finished testing {PackageId} {PackageVersion}", packageId, packageVersion);
+                foreach (var testResult in (TestResult[])Enum.GetValues(typeof(TestResult)))
+                {
+                    var validationsWithResult = testResults.EndpointValidationResults.SelectMany(r => r.ValidationResults).Where(r => testResult == r.Result);
+                    if (validationsWithResult.Any())
+                    {
+                        foreach (var validationTestResult in validationsWithResult)
+                        {
+                            var testResultLogString = "{PackageId} {PackageVersion}: {ValidationTest}: {TestResult}";
+                            var validationTestName = validationTestResult.ValidationTest.GetType().Name;
+                            var testResultString = testResult.ToString();
+
+                            switch (testResult)
+                            {
+                                case TestResult.Pass:
+                                    _logger.LogInformation(testResultLogString, packageId, packageVersion, validationTestName, testResultString);
+                                    break;
+                                case TestResult.Skip:
+                                    _logger.LogWarning(testResultLogString, packageId, packageVersion, validationTestName, testResultString);
+                                    break;
+                                case TestResult.Fail:
+                                    _logger.LogError(testResultLogString + " - {Exception}", packageId, packageVersion, validationTestName, testResultString, validationTestResult.Exception);
+                                    break;
+                            }
+                        }
+                    }
+                }
+            }
+            catch (Exception e)
+            {
+                _logger.LogError("Failed to test {PackageId} {PackageVersion}! {Exception}", packageId, packageVersion, e);
```

Comment:
Our jobs infrastructure currently doesn't support logging to anything other than `ILogger`s...changing that would require a nontrivial refactoring.

We only use `TelemetryClient` in our MVC/Owin services.